examples/node-app-pg/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "app-pg",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint src/index.js",
+    "lint:fix": "eslint --fix src/index.js",
+    "sync-db": "sync-db",
+    "start": " babel-node src/index.js",
+    "make": "knex --knexfile=knexfile.js migrate:make",
+    "migrate": "knex --knexfile=knexfile.js migrate:latest",
+    "rollback": "knex --knexfile=knexfile.js migrate:rollback"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@leapfrogtechnology/sync-db": "^1.0.0-alpha.6",
```

Use the latest version.
```suggestion
    "@leapfrogtechnology/sync-db": "^1.0.0-alpha.8",
```